### PR TITLE
Add link to work in card and translate category even in mosaic

### LINKED ISSUE #472: Ajouter un bouton (i) pour accéder à la fiche du Work (ou en cliquant sur le titre)
Parce que ça nous saoule de scroller.

### PR DESCRIPTION
**Pull request by jilljenn**
Fixes #472.

### CODE REVIEW

File: mangaki/mangaki/static/js/vote.js [hunk 1]
```diff
@@ -138,11 +138,14 @@ Card.prototype.hydrate = function (work) {
 
   // FIXME: We should use a solution that doesn't require monkey-patching HTML
   // with jQuery in the future, but for now that is the easiest way.
+  var card_url = '/' + this.category + '/' + work.id;
```
Comment (RaitoBezarius) [inline mangaki/mangaki/static/js/vote.js]: Please use `const` rather than `var`

File: mangaki/mangaki/static/js/vote.js [hunk 2]
```diff
@@ -138,11 +138,14 @@ Card.prototype.hydrate = function (work) {
...
+  console.log(work);
   $el.find('.work-card__media').css('background-image', 'url("' + work.poster + '")');
-  $el.find('.work-card__title').text(work.title);
-  $el.find('.work-cover__wrapper').attr('data-category', work.category);
+  $el.find('.work-card__title').html('<a href="' + card_url + '">' + work.title + '</a>');
```
Comment (RaitoBezarius) [inline mangaki/mangaki/static/js/vote.js]: Don't you want to select the inner `<a />` element and change its attribute `href`, then, change its text, something along `$el.find('.work-card__title').find('a')` would work surely, using a better selector, you could get it in one shot. My selector skills are a bit rusty, something like: `a < .work-card__title` could work, I don't remember.

File: mangaki/mangaki/static/js/vote.js [hunk 3]
```diff
@@ -138,11 +138,14 @@ Card.prototype.hydrate = function (work) {
 
   // FIXME: We should use a solution that doesn't require monkey-patching HTML
   // with jQuery in the future, but for now that is the easiest way.
+  var card_url = '/' + this.category + '/' + work.id;
+  console.log(work);
```
[the remark this answers is not in the document]
Comment (jilljenn) [reply mangaki/mangaki/static/js/vote.js]: Done!

File: mangaki/mangaki/static/js/vote.js (hunk 1 above)
Comment (jilljenn) [reply mangaki/mangaki/static/js/vote.js]: Done!

**Pull request merged**